generator/js2yaml.html

```diff
@@ -16,16 +16,18 @@
 <h1>Convert JS examples into Yaml</h1>
 
 <div>
-	<label for="input">Input JS code:</label>
-	<textarea id="input">
+    <label for="input">Input JS code:</label>
+    <textarea id="input">
 [
   // Tests
   {
     $project: {
       date: new Date(36),
-	  regex: /^123/,
-	  regex_with_flags: /^123/i,
-	  binary: BinData(0, "ID=="),
+      regex: /^123/,
+      regex_with_flags: /^123/i,
+      binary: BinData(0, "ID=="),
```

Comment:
```suggestion
      binary: BinData(0, "IA=="),
```

May as well use the canonical value (per [DOCSP-35733](https://jira.mongodb.org/browse/DOCSP-35733)).